workspaces/frontend/src/main.ts

```diff
@@ -225,6 +224,7 @@ export const init = (
         AI_MAX_COLLISION_DODGE_ERROR: 0.75, // seconds
         AI_COLLISION_DODGE_ERROR_DECREASE_SPEED: 0.33,
         AI_MAX_COLLISION_PANIC_RADIUS: 20, // px
+        AI_ANGLE_DIFF_THRESHOLD: (fps / -30 + 4), // returns 3 for 30FPS and 2 for 60FPS
```

Comment:
Weird but ok

Reply:
I wanted to put in there an expression if we ever wanted to try other FPS values...